src/io/io.c

```diff
@@ -434,3 +434,57 @@ int MVM_set_std_handle_to_nul(FILE *file, int fd, BOOL read, int std_handle_type
     return 1;
 }
 #endif
+
+char* MVM_process_path(MVMThreadContext *tc, MVMString *path) {
```

Comment:
I assume I should rename this. Any suggestions?

Reply:
Heh, I was thinking the same thing. Don't have any great ideas yet.

Reply:
MVM_platform_path?